hardhat.config.ts

```diff
@@ -24,7 +24,7 @@ task("accounts", "Prints the list of accounts", async (args, hre) => {
 
 const config: HardhatUserConfig = {
   solidity: {
-    version: "0.8.7",
+    version: "0.8.17",
```

Comment:
Have you considered v0.8.19 that is already available?

Reply:
Yes but hardhat still doesn't support it, unfortunately. Once it does we plan on upgrading 💪